linera-execution/src/wasm/wasmer.rs

```diff
@@ -12,11 +12,8 @@ use linera_witty::{
     ExportTo, Instance,
 };
 use tokio::sync::Mutex;
-use wasmer::{
-    sys::EngineBuilder, wasmparser::Operator, CompilerConfig, Cranelift, Engine, Module,
-    Singlepass, Store,
-};
-use wasmer_middlewares::metering::{self, Metering, MeteringPoints};
+use wasm_instrument::{gas_metering, parity_wasm};
+use wasmer::{sys::EngineBuilder, Cranelift, Engine, Module, Singlepass, Store};
```

Comment:
Question: is Wasmtime support not part of the plan?

Reply:
It is! But hoisting this up a level is for after the Web wallet is working :)